client/src/components/HistoryViewer/HistoryViewer.scss

```diff
@@ -22,11 +22,10 @@
   margin-left: 10px;
 }
 
-.history-viewer__table--current {
+.history-viewer__row--current {
   // There will only be one table row in this table state
-  tbody tr,
-  tbody tr:hover {
-    background-color: $link-color;
+  &, &:hover {
+    background-color: $link-color !important;
```

Comment:
Please don't use `!important` - make the CSS more specific instead. If it's necessary in order to override another `!important` higher up can you add a comment to say that?

Reply:
Yeah I felt bad when I did this... I'll be more specific, but I'll have to go right back to `table tbody tr.blah`.

Reply:
This is why we need BEM everywhere!

Reply:
Ok done. Wasn't as bad as I thought.